spring-cloud-stream-binders/spring-cloud-stream-binder-kafka/src/test/java/org/springframework/cloud/stream/binder/kafka/KafkaBinderTests.java

```diff
@@ -314,16 +315,15 @@ public void testCustomPartitionCountOverridesPartitioningIfLarger() throws Excep
 
 		byte[] ratherBigPayload = new byte[2048];
 		Arrays.fill(ratherBigPayload, (byte) 65);
-		KafkaTestBinder binder = (KafkaTestBinder) getBinder();
+		KafkaTestBinder binder = getBinder();
 
 		DirectChannel moduleOutputChannel = new DirectChannel();
 		QueueChannel moduleInputChannel = new QueueChannel();
-		Properties producerProperties = new Properties();
-		producerProperties.put(BinderPropertyKeys.MIN_PARTITION_COUNT, "5");
-		producerProperties.put(BinderPropertyKeys.NEXT_MODULE_COUNT, "3");
-		producerProperties.put(BinderPropertyKeys.PARTITION_KEY_EXPRESSION, "payload");
-		Properties consumerProperties = new Properties();
-		consumerProperties.put(BinderPropertyKeys.MIN_PARTITION_COUNT, "5");
+		KafkaProducerProperties producerProperties = new KafkaProducerProperties();
+		producerProperties.setPartitionCount(5);
```

Comment:
similar question here... it seems as though the intent of these 2 tests was to verify overriding of the partition count based on these 2 different properties... but now, if there are no longer 2 different properties, do we even need these tests?